README.md

```diff
@@ -1,55 +1,127 @@
 # sie-backend
 
-The repository contains code for backend infrastructure of Self Image Experiment (SIE) web-application using Google Cloud Platform. The following sections below will provide you more information on how to setup the local environment.  
-
-
+The repository contains code for backend infrastructure of Self Image Experiment (SIE) web-application using Google Cloud Platform. The following sections below will provide you more information on how to setup the local environment.
 
 ## Setup development environment
 
 To setup your local env, you will need to install Python 3.6+ and Google SDK in a UNIX based operating system.
 
-* ### Clone the repository 
+- ### Clone the repository
 
-    ```
-    $ git clone https://github.com/neu-self-image-experiments/sie-backend.git
-    ```
+  ```
+  $ git clone https://github.com/neu-self-image-experiments/sie-backend.git
+  ```
 
-* ### Installing dependencies
-    If you do not have `venv` package you can first install it:
-    ```
-    $ sudo apt install python3-venv
-    ```
-    Then to create new virtual environment run the following command in your project directory:
-    ```
-    $ sudo python3 -m venv venv
-    ```    
-    You need to activate the virtual environment in order to use it. So activate the virtual environment using the following command:
-    ```
-    $ source venv/bin/activate
-    ```
-    Once activated, you would see the shell prompt will change and it will show the name of the virtual environment that you are currently using. In our case it is venv.
+- ### Installing dependencies
 
-    Now install all the dependencies using:
-    ```
-    (venv) $ sudo pip3 install -r requirements.txt
-    ```
+  If you do not have `venv` package you can first install it:
 
-    Now run the flask server:
-    ```
-    (venv) $ python3 main.py
-    ```
+  ```
+  $ sudo apt install python3-venv
```

Comment:
nit: this is only for linux, right? I'm curious but can we run `python3 -m venv <name>` with just python3?

Reply:
Sometimes ppl don't have access set up. So to be on safe side I wrote sudo. But I can get rid of it.